### PR TITLE
feat: add new subpage whichHufflepuffAreYou

### PR DESCRIPTION
**Pull request by Ervige**

### CODE REVIEW

File: pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    font-family: var(--titleFont);
+}
+
+.housePride__category {
+    margin: 15px 0;
+    font-family: var(--mainfont);
+}
+
+.housePride__underline {
+    font-family: var(--mainfont);
```
Comment (lukasz-rdzanek) [inline pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css:41]: In code you have two variables "--mainfont" and in other place "--mainFont". Which one is correct? Perhaps this with camelCase.

File: pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.housePride__headline {
+    font-size: 28px;
+    font-family: var(--titleFont);
+}
+
+.housePride__category {
+    margin: 15px 0;
+    font-family: var(--mainfont);
```
Comment (lukasz-rdzanek) [inline pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css:37]: same as below

File: pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.writtenBy__line {
+    width: 40%;
+    color: var(--gray);
+    padding: 0 30px;
+}
+
+.writtenBy__title {
+    font-family: var(--mainfont);
```
Comment (lukasz-rdzanek) [inline pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css:126]: same as above

**Review by lukasz-rdzanek — CHANGES_REQUESTED** [review]
A few changes are required

File: pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css (hunk 1 above)
Comment (Ervige) [reply pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css]: @lukasz-rdzanek resolved

File: pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css (hunk 3 above)
Comment (Ervige) [reply pages/housePrideQuiz/hufflepuff/whichHufflepuffAreYou.css]: @lukasz-rdzanek resolved

**Review by lukasz-rdzanek — APPROVED** [review]

**Pull request merged**